From 70ecfc71c7bf6400560b964d8ed9dfaf2a6abb57 Mon Sep 17 00:00:00 2001
From: Matthias Klumpp <matthias@tenstral.net>
Date: Thu, 6 Aug 2015 15:18:02 +0200
Subject: [PATCH 1/2] Add AppStream metadata

BUG:352607

(cherry picked from commit fa2014c9d3edb2cb144bc9f5f0edafa789c2975c)

Conflicts:
	umbrello/CMakeLists.txt
---
 umbrello/org.kde.umbrello.appdata.xml | 32 ++++++++++++++++++++++++++++++++
 1 file changed, 32 insertions(+)
 create mode 100644 umbrello/org.kde.umbrello.appdata.xml

diff --git a/umbrello/org.kde.umbrello.appdata.xml b/umbrello/org.kde.umbrello.appdata.xml
new file mode 100644
index 0000000..bdac08d
--- /dev/null
+++ b/umbrello/org.kde.umbrello.appdata.xml
@@ -0,0 +1,32 @@
+<?xml version="1.0" ?>
+<component type="desktop">
+  <id>org.kde.umbrello.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0+</project_license>
+  <name>Umbrello</name>
+  <summary>UML Modeller</summary>
+  <description>
+    <p>
+      Umbrello is a Unified Modelling Language (UML) modelling tool and code generator.
+      It can create diagrams of software and other systems in the industry-standard UML format, and can also generate code from UML diagrams
+      in a variety of programming languages.
+    </p>
+    <p>Features:</p>
+    <ul>
+      <li>Supported formats: XMI</li>
+      <li>Several type of diagrams supported: use case, class, sequence, collaboration, state, activity, component, deployment, entity relationship</li>
+    </ul>
+  </description>
+  <url type="homepage">http://umbrello.kde.org</url>
+  <url type="bugtracker">https://bugs.kde.org/enter_bug.cgi?format=guided&amp;product=umbrello</url>
+  <url type="help">http://docs.kde.org/stable/en/kdesdk/umbrello/index.html</url>
+  <screenshots>
+    <screenshot type="default">
+      <image>http://kde.org/images/screenshots/umbrello.png</image>
+    </screenshot>
+  </screenshots>
+  <project_group>KDE</project_group>
+  <provides>
+    <binary>umbrello</binary>
+  </provides>
+</component>
-- 
2.4.1

